WORKSPACE

```diff
@@ -23,3 +23,8 @@ envoy_python_dependencies()
 load("//bazel:dependency_imports.bzl", "envoy_dependency_imports")
 
 envoy_dependency_imports()
+
+load("//tools/protojsonschema:com_github_chrusty_protoc_gen_jsonschema_deps.bzl", "go_dependencies")
+
+# gazelle:repository_macro //tools/protojsonschema/com_github_chrusty_protoc_gen_jsonschema_deps.bzl%go_dependencies
+go_dependencies()
```

Comment:
there shouldnt be any changes here - the naming is not great, but check the order of existing loads and use them